migrations/versions/d2733490ad16_.py

```diff
@@ -0,0 +1,41 @@
+"""empty message
```

Comment:
Can you add a message here? It makes it easier when looking at the migration history.

Reply:
👍